constant.py

```diff
@@ -1,3 +1,9 @@
+#class name for electricain name
```

Comment:
Constants should be named in upper case. 

contactclass is incorrect way of naming, in python if your name has two words you separate it with an underscore.

Reply:
Still incorrect, it should be fully uppercase, this is title case